src/app/clusters/administrator-commissioning-server/administrator-commissioning-server.cpp

```diff
@@ -36,6 +38,7 @@
 #include <setup_payload/SetupPayload.h>
 #include <system/SystemClock.h>
 #include <tracing/macros.h>
+#include <zap-generated/gen_config.h>
```

Comment:
This should be including config.h, not gen_config.h.  gen_config.h does not exist in all configurations.

Reply:
I've created https://github.com/project-chip/connectedhomeip/pull/37464 to address this.